opentracing-spring-cloud/src/main/java/io/opentracing/contrib/spring/cloud/async/ExecutorBeanPostProcessor.java

```diff
@@ -0,0 +1,120 @@
+package io.opentracing.contrib.spring.cloud.async;
+
+/**
```

Comment:
Just wondering why the author annotation, and copyright header are in unusual positions.

Reply:
It will be modified I am waiting for the response from legal